### ISSUE TITLE
ld: framework not found HockeySDK

### ISSUE THREAD

**Issue by preneond** [bitstadium/HockeySDK-iOS#367]
After migrating to swift 3 I am having problem with HockeySDK pod. Few hours ago everything worked well but now I am unable to build it. I am getting Match-o  linker error, that framework wasn't found. I am having the same issue, when I work with previous commits, where everything was ok. The same issue i am having also after deleting derived data and restarting the system. Thanks for answer!
<img width="1440" alt="screen shot 2016-11-08 at 10 06 40" src="https://cloud.githubusercontent.com/assets/17913722/20093379/c9f6d062-a59c-11e6-8714-e550f27adca6.png">

**Comment by TroubleMakerBen** [comment]
Hey @preneond 

thx for getting in touch.
I'm not aware of any issue with our pod and just integrated HockeySDK 4.1.2 into a project with Xcode 8.1, Cocoapods 1.1.1 without any issues.

Which version of our SDK are you using?

Feel free to open a ticket in our [support](https://support.hockeyapp.net) to share more information privately.

Best,
Benjamin

**Comment by madhusamuel** [comment]
I too started to get the exact same error all of a sudden on xcode 8.1 and swift 3. One day before it used to build properly.

I upgraded cocoapods to 1.1.0. Deleted the pods directory and Podfile.lock and did a fresh pod install. Magically the "framework not found HockeySDK" error just vanished.

**Comment by TroubleMakerBen** [comment]
Hi guys,
Glad you guys could resolve the issue. It's always a good idea to have an updated cocoapods version if you're switching to a newer Xcode version.

Have a nice weekend!

Benny

PS: Closing, looks like it's an duplicate of #363 and #365.

**Comment by preneond** [comment]
few days later it has started to work normally, like nothing happened. It's just  weird, but thanks a lot for support! 👍

**Comment by dmdeller** [comment]
TLDR: To fix, delete `Pods` folder, and re-run `pod install`.

Our team ran into the same issue, also after a Swift 3 / Xcode 8.1 upgrade. Cocoapods was already at v1.1.1 and re-running `pod install` did not resolve the issue. (Also tried the usual dance of cleaning DerivedData etc.).

After reading this thread, I deleted the `Pods` folder, but left `Podfile.lock` as-is. Then I re-ran `pod install` and tried building the project again in Xcode. The build succeeded.

Git showed that a binary file `Pods/HockeySDK/HockeySDK-iOS/HockeySDK.embeddedframework/HockeySDK.framework/HockeySDK` had changed, but there were no other changes. (Strangely enough, this change only showed up in command line `git diff`, not in Tower.) I committed the change and pushed it to my team, which also solved the problem for them.

This seems like a possible bug in CocoaPods and/or Xcode, but I'm posting my results here in case this helps someone else.

**Comment by lumaxis** [comment]
Thanks for the info @dmdeller !

**Comment by qiulang** [comment]
I think @dmdeller is correct. The changed HockeySDK file caused the problem. Our .gitignore ingores binary file so when others checkout my codes they have this linker error.

**Comment by StupidDutchName** [comment]
I suspect the problem here is one of capitals. I deleted my `Pods` folder, but git saw no changes... So it works on my local machine, but not on Xcode Server. Problem is probably capitals, so easiest solution is to clean all caches on Xcode Server, or commit your project with the `Pods` folder removed, let your CI / other machines check this out (and fail), then run `pod install` and commit / push that later.
